stable/kong/templates/deployment.yaml

```diff
@@ -251,5 +251,5 @@ spec:
       volumes:
         - name: custom-nginx-template-volume
           configMap:
-            name: {{ .Release.Name }}-kong-default-custom-server-blocks
+            name: {{ template "kong.fullname" . }}-default-custom-server-blocks
```

Comment:
Why is this changed?

Reply:
It may not be the matter of this PR, but in general, all objects in a chart should be addressing `{{ template "<chartname.fullname>" }}` in its names.

It allows a user to override objects naming for the whole installation, by taking into account `.Values.nameOverride` and `.Values.fullNameOverride` (not yet in Kong chart) variables passed to the chart.